packages/pieces/community/google-drive/src/lib/action/add-permission.action.ts

```diff
@@ -70,7 +70,8 @@ export const addPermission = createAction({
         const result = await drive.permissions.create({
             requestBody: permission,
             fileId: fileId,
-            sendNotificationEmail: send_invitation_email
+            sendNotificationEmail: send_invitation_email,
+            supportsAllDrives: true,
```

Comment:
do you need to wrap this in a toggle? i'm not sure what's the impact of hardcoding this but in almost all other places this is toggleable

https://github.com/search?q=repo%3Aactivepieces%2Factivepieces+path%3A%2F%5Epackages%5C%2Fpieces%5C%2Fcommunity%5C%2Fgoogle-drive%5C%2Fsrc%5C%2Flib%5C%2F%2F+supportsAllDrives&type=code